src/http.ts

```diff
@@ -1,3 +1,48 @@
+import {
```

Comment:
Just moved these imports to the top, to keep all the type imports grouped at the bottom.